tests/test_api_urls.py

```diff
@@ -71,6 +71,19 @@ def test_api_labels_without_auth(self):
         self.assertEqual(rv.status_code, 200)
         self.assertEqual(rv.content_type, 'application/json')
 
+    def test_api_comments_link_header_auth(self):
+        '''API access to comments greater than 30 returns pagination in Link header of the response.'''
+        query_string = {'callback': 'foo'}
```

Comment:
Can you help me understand that? 

https://github.com/webcompat/webcompat.com/blob/master/webcompat/api/endpoints.py#L201 doesn't take `callback` arguments, and the GitHub comments API doesn't have any `callback` params: https://developer.github.com/v3/issues/comments/

What is consuming this callback param?